jit-compiler/src/compiler.rs

```diff
@@ -184,6 +184,9 @@ name = "powdr_jit_compiled"
 version = "0.1.0"
 edition = "2021"
 
+[profile.release]
+opt-level = 1
```

Comment:
With this, the time to compute fixed columns is about the same (~40s for the Keccak Rust example).